### PR TITLE
Fix some runtime errors when running osu.iOS on iPhoneSimulator

### PR DESCRIPTION
**Pull request by smoogipoo**
Still doesn't work yet - fails with some DbContext ctor error, some mismatch is happening somwhere in Microsoft.Extensions.Logging/Microsoft.Extensions.Logging.Abstractions.

Hoping Samah can some in and fix it :P

### CODE REVIEW

File: osu.iOS/Main.cs [hunk 1]
```diff
@@ -9,6 +9,9 @@ public class Application
         // This is the main entry point of the application.
         static void Main(string[] args)
         {
+            // required to initialise native SQLite libraries on some platforms.
+            SQLitePCL.Batteries_V2.Init();
```
Comment (peppy) [inline osu.iOS/Main.cs]: isn't this already done in the DbContexts?

Comment (smoogipoo) [reply osu.iOS/Main.cs]: The netstandard branch moved this out of DbContext :/

Comment (peppy) [reply osu.iOS/Main.cs]: Hmm i see.. so where is it on desktop?

Comment (peppy) [reply osu.iOS/Main.cs]: Also why can it not be in DbContext?

Comment (smoogipoo) [reply osu.iOS/Main.cs]: https://github.com/ppy/osu/blob/ios/osu.Desktop/Program.cs#L19

I think I moved it to initially reduce references in osu.Game when redoing the csproj's, should be possible to move it back.

**Pull request merged**